SearchManager.h

```diff
@@ -0,0 +1,37 @@
+#ifndef TICTACTOE_SERVER_SEARCHMANAGER_H
+#define TICTACTOE_SERVER_SEARCHMANAGER_H
+
+#include <unordered_map>
+#include <random>
+
+class SearchManager {
+public:
+	struct PlayerInfo {
+		std::string id;
+		std::string adress;
+	};
+
+private:
+	unordered_map<unsigned, PlayerInfo> active_searches;
```

Comment:
Same error as before. No class/struct/union with this name in the current namespace